File: patient-management-app/src/app/Model/patient.ts
export class Patient {

    id: string;
    firstName: string;
    lastName: string;
    email: string;
    phoneNumber: string;
    dateBirth: any;
  
    constructor(id: string, firstName: string, lastName: string, email: string, phoneNumber: string, dateBirth: any) {
      this.id = id;
      this.firstName = firstName;
      this.lastName = lastName;
      this.email = email;
      this.phoneNumber = phoneNumber;
      this.dateBirth = dateBirth;
    }
}

File: api-patient-management/api-patient-management/Models/Patient.cs
using System;
using System.ComponentModel.DataAnnotations;
using api_patient_management.CustomValidation;

namespace api_patient_management.Models
{
	public class Patient
	{

		public Guid Id { get; set; }
      
        [StringLength(50, MinimumLength = 2)]
        public string? FirstName { get; set; }
      
        public string? LastName { get; set; }
   
        public string? Email { get; set; }
        [Phone]
        public long PhoneNumber { get; set; }
        [DataType(DataType.Date)]
        [DisplayFormat(DataFormatString = "{0:yyyy-MM-dd}", ApplyFormatInEditMode = true)]
        [Display(Name = "My Date")]
        [DateNotInFuture]
        public DateTime? DateBirth { get; set; }

    }

   
}


